docs/LocalDevelopmentGuide.md

```diff
@@ -175,6 +175,9 @@ Request URL, submit the ngrok HTTPS URL with `/slack/commands` appended to the
 end. For the actual command, anything will work, though the final app will use
 `/rocket`.
 
+Select the checkbox titled "Escape channels, users, and links sent to your app"
```

Comment:
Shouldn't the command be under a new section like 7.4?

Reply:
It's in the same dialog box, so probably most likely no.